nntrainer/compiler/ini_interpreter.h

```diff
@@ -0,0 +1,81 @@
+// SPDX-License-Identifier: Apache-2.0
+/**
+ * Copyright (C) 2021 Jihoon Lee <jhoon.it.lee@samsung.com>
+ *
+ * @file ini_interpreter.h
+ * @date 02 April 2021
+ * @brief NNTrainer Ini Interpreter
+ * @see	https://github.com/nnstreamer/nntrainer
+ * @author Jihoon Lee <jhoon.it.lee@samsung.com>
+ * @bug No known bugs except for NYI items
+ */
+#include <iostream>
+#include <memory>
+
+#include <iniparser.h>
```

Comment:
I wonder that model loader also use iniparser to parse. Is it to save graph to ini ?? or read ini? If it is to read ini then.. I guess better to use model_loader.

Reply:
Yes this is duplication so reading graph part will move here and related function call will be removed. https://github.com/nnstreamer/nntrainer/pull/1085